_docs/reference/notes/index.md

```diff
@@ -1,11 +1,13 @@
 ---
-title: Prior Release Notes
-overview: Release notes for prior versions of Istio.
+title: Release Notes
+overview: What's been happening with Istio.
 
-order: 600
+order: 10
```

Comment:
this puts it after API, it should probably be second below Feature Status (or even first)

![screen shot 2017-12-05 at 11 49 05 am](https://user-images.githubusercontent.com/3664595/33627420-51aac956-d9b2-11e7-8ebf-68702d54ba82.png)

Reply:
I moved it above API. At the moment, "directories" always come after plain files, so that's as high as I can make it.